src/pages/view-test-result/components/test-details-card/test-details-card.ts

```diff
@@ -16,4 +16,8 @@ export class TestDetailsCardComponent {
     return specialNeedsArray.length > 0 && specialNeedsArray[0] !== 'None';
   }
 
+  showFullCatHeld(): boolean {
+    return this.data.category.includes('+E');
```

Comment:
This is a common component so we need to be more specific here. This would currently display the field for B+E which we do not want. Should do something like

```return this.data.category === TestCategory.CE || this.data.category === TestCategory.C1E || this.data.category === TestCategory.DE || this.data.category === TestCategory.D1E;```

Or if got a reservation about the OR's, then something like

```return [TestCategory.CE, TestCategory.C1E, TestCategory.DE, TestCategory.D1E].some((category) => this.data.category);```

Reply:
might be worth making a helper function to identify these categories as its likely to come up again

Reply:
@andrewsetterfield There are helpers in the doc service so this is something we could introduce into the FE. But lets leave that for another ticket .

FYI - https://github.com/dvsa/mes-documents-service/blob/develop/src/functions/sendCandidateResults/application/service/template-id-provider.ts

Reply:
was getting a typing error when using .some() so for convenience i've done it with the inverse use of .includes() instead.

Tested it and works as required but let me know if there's any reasons not to use this and to stick with using .some()